src/components/PopupWithForm.js

```diff
@@ -6,6 +6,7 @@ export default class PopupWithForm extends Popup {
     this._handleFormSubmit = handleFormSubmit;
     this._formElement = this._popupElement.querySelector(".modal__form");
     this._inputElements = this._popupElement.querySelectorAll(".modal__input");
+    this._submitButton = this._popupElement.querySelector(".modal__button");
   }
```

Comment:
**COULD BE IMPROVED**

You can make a special method for getting a form element from `PopupWithForm` not to **search** it in `index.js`



```js

  getForm() {
    return this._formElement
  }


```